Correct example code for OES_get_program_binary

tombsar opened the pull request:
`glGetProgramBinaryOES` should take the binary size by value, but a pointer was passed in the example code.

Perksey commented:
> Actually, can you bump the version number and add a log to the revision history.

A fair few of the recent typo fixes of this kind have not followed that procedure. For future reference is that something that's supposed to be done for every spec change? (would be good to know as I look at every PR and will be able to flag it up early if it's meant to be done)

tombsar commented:
Would it be appropriate to add myself to the list of contributors, or is that reserved for people who made more meaningful contributions to the document?

pdaniell-nv commented:
> Would it be appropriate to add myself to the list of contributors, or is that reserved for people who made more meaningful contributions to the document?

Yes, sure, add yourself there.

tombsar commented:
I've bumped the version number and updated the revision log (and added my name to the contributors list) as requested.

Not sure what date I should have put. The date the changes get merged seems most appropriate to me, but that's outside my control.

pdaniell-nv commented:
> > Would it be appropriate to add myself to the list of contributors, or is that reserved for people who made more meaningful contributions to the document?
> 
> Yes, sure, add yourself there.

Sorry I misunderstood your question. I thought you meant in the revision log. I tweaked your change.

tombsar commented:
> Sorry I misunderstood your question.

No worries; I thought it was too much to hope for! ;)
